util/wrap.pl.in

```diff
@@ -22,7 +29,10 @@
 my $use_system = 0;
 my @cmd;
 
-if (-x $unix_shlib_wrap) {
+if ($^O eq 'VMS') {
+    # VMS needs the command to be appropriately quotified
+    @cmd = quotify_cmd(@ARGV);
+} elsif (-x $unix_shlib_wrap) {
     @cmd = ( $unix_shlib_wrap, @ARGV );
 } else {
     # Hope for the best
```

Comment:
Note: I'm unsure if the command line fixup shouldn't be done here too...  Maybe Windows needs it as well, and we've just been damn lucky that we never encountered a problematic command line?